homework-submission/week2/src/ToDo/DoneCheckBox.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+let isChecked = false;
+
+const todos = [
+  {
+    "id": 1,
+    "description": "Get out of bed",
+    "deadline": "2017-09-11",
+    "done": true
+  },
+  {
+    "id": 2,
+    "description": "Brush teeth",
+    "deadline": "2017-09-10",
+    "done": false
+  },
+  {
+    "id": 3,
+    "description": "Eat breakfast",
+    "deadline": "2017-09-09",
+    "done": false
+  }
+]
+
+export default class DoneCheckBox extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { isChecked: this.props.done };
+    this.handleCheck = this.handleCheck.bind(this);
+  }
+
+  /*componentWillMount() {
+    this.setState(() => (
+      {
+        isChecked: false//todos[this.props.id].done,
+      }
+    ));
+  }*/
+
+  handleCheck = () => {
+    console.log('----------' + isChecked);
+    this.setState({
+      isChecked: !this.state.isChecked
+    })
+    /*this.setState(() => (
+      {
+        isChecked: !isChecked,
+      }
+    ));*/
+    console.log('++++++++++' + isChecked);
+    console.log('++++++++++' + !this.state.isChecked);
+  }
```

Comment:
this function should also be created in the App and pass it here as props to change the state in the App